liquidctl/driver/commander_pro.py

```diff
@@ -245,12 +258,15 @@ def initialize(self, direct_access=False, **kwargs):
         if self._hwmon and not direct_access:
             _LOGGER.info('bound to %s kernel driver, assuming it is already initialized',
                          self._hwmon.driver)
+
+            if fan_modes:
+                _LOGGER.warning('kernel driver does not support the `fan mode` options, ignoring')
```

Comment:
FYI I might still tweak this message, for consistency with other hwmon drivers and docs/developer/hwmon.md.